src/Pic.vue

```diff
@@ -1,13 +1,15 @@
 <template>
     <div class="image-wrapper">
-        <img :src="src" :alt="alt" :width="width" :height="height">
+        <img ref="srcImg" @load.once="setWidth" :src="src" :alt="alt" :width="[hasWidth ? width : (hasHeight ? localWidth : '')]" class="img-fluid rounded">
```

Comment:
* `"srcImg"` -> `"img"` would probably be sufficient (following the rest of the codebase).
* Instead of the logic in `width`, perhaps just always update `localWidth`, and use that instead? In other words, maybe this logic can just be in `computeWidth()`.
* Is it possible to hook into the lifecycle (e.g. `mounted`) rather than relying on `@load.once`?

Reply:
- Mounted lifecycle hook with `this.$nextTick` does not work, the browser has yet to fetch the img from the URL.